components/director/internal/domain/label/repository.go

```diff
@@ -174,6 +180,56 @@ func (r *repository) DeleteByKey(ctx context.Context, tenant string, key string)
 	return nil
 }
 
+func (r *repository) GetRuntimesIDsByKeyAndValue(ctx context.Context, tenantID, key, value string) ([]string, error) {
```

Comment:
What do you think about this name: 
```suggestion
func (r *repository) GetRuntimesIDsByStringLabel(ctx context.Context, tenantID, key, value string) ([]string, error) {
```